src/lib/journey/callbacks/password/confirm-input.svelte

```diff
@@ -41,9 +43,18 @@
   }
 
   $: {
+    isInvalid =
+      passwordsDoNotMatch || (isRequired && typeof value === 'string' && value.length === 0);
```

Comment:
I believe the second portion of this condition isn't necessary, now that I think about it. This shouldn't be the only way an input gets marked as invalid. The `form.svelte` component can also mark inputs as invalid. So, I'd recommend you at least attempt to only using `passwordsDoNotMatch` as the value for `isInvalid` and see if everything still works. In other words, remove the `|| (isRequired && typeof value === 'string' && value.length === 0)` from this line.